security/us-east-1/security-compliance --/awsconfig.tf

```diff
@@ -6,23 +6,32 @@
 # AWS Config Logs AES256 SSE Bucket
 #
 module "config_logs" {
```

Comment:
@diego-ojeda-binbash One thing to consider, renaming this will complaint in current deployments. We need to warn that this module will changed and include a note to move the resource in the tfstate. 

I would include this change in another iteration.